src/CachePlugin.php

```diff
@@ -34,31 +35,86 @@
     private $config;
 
     /**
+     * Cache directives indicating if a response can not be cached.
+     *
+     * @var array
+     */
+    private $noCacheFlags = ['no-cache', 'private'];
```

Comment:
right. lets open an issue to implement the full no-cache behaviour in the future. but rather than completely ignoring, completely not caching is at least correct according to the protocol.